src/ucp/wireup/wireup_cm.c

```diff
@@ -357,13 +357,33 @@ ucp_cm_ep_priv_data_pack(ucp_ep_h ep, const ucp_tl_bitmap_t *tl_bitmap,
     return status;
 }
 
+void ucp_wireup_cm_ep_pending_extract(ucp_ep_t *ucp_ep, ucs_queue_head_t *queue)
+{
+    const ucp_lane_index_t lane_idx = ucp_ep_get_cm_lane(ucp_ep);
```

Comment:
Can pass `lane` as an arg (it exists in the calling scope)

Reply:
I thought to simplify the function definition and also make it more general, can be called when lane is not in the scope of calling scope (At the moment it's in the scope but maybe it will be used in the future in more places).

Reply:
IMO better to avoid a redundant call to `ucp_ep_get_cm_lane`, @evgeny-leksikov WDYT?

Reply:
After another review and adding the helper var - I agree. I've change it, less calls and looks better.